### ISSUE TITLE
Create a /collections page to show more details about collections

### ISSUE THREAD

**Issue by alohr51** [kabanero-io/kabanero-website#2]
**This work is on hold until we have more collections / details**

Create collections page to show off more of our collections. We still want to show featured collections on the homepage.

We currently do not have a need for a /collections page, but as we get more collections and details added we will want this.

See [Melissa's comment below](https://github.com/kabanero-io/kabanero-website/issues/2#issuecomment-515186775) for the design mockup/redlines.

**Comment by kSee04** [comment]
Putting this in the backlog for now pending design work.

**Comment by marikaj123** [comment]
Needs more design work and implementation.

**Comment by marikaj123** [comment]
For release August 12

**Comment by marikaj123** [comment]
Assignee: Melissa

**Comment by marikaj123** [comment]
According to Melissa work on track

**Comment by marikaj123** [comment]
The dev team is waiting on the wireframe work to complete before finishing the implementation work.

**Comment by mdenby** [comment]
Here is a simple Collections page. More collections will be added to this page over time.

1. Add Collections to the navigation.
2. Basically copy the collections section from the homepage.

Here are the redlines. Please let me know if you have any problems with them.
[Collections redlines.zip](https://github.com/kabanero-io/kabanero-website/files/3433021/Collections.redlines.zip)

![Collections](https://user-images.githubusercontent.com/26823018/61903452-877dd480-aef2-11e9-95e8-3d49c0159e7e.jpg)

**Comment by alohr51** [comment]
@mdenby is this good to go? I see you said copy, but I want to double check if we _moving_ the section from the homepage or _copying_ it to this collections page.

**Comment by mdenby** [comment]
Let's say _copying_ for now...last time I said move and Nate said to put it back lol

**Comment by alohr51** [comment]
haha okay, I am unsure about having the exact same content in 2 places on the site. @nateziemann are you okay with us _moving_ the "Kabanero Collections" section from the homepage to a `/collections` page?

**Comment by nateziemann** [comment]
I want to advertise at least 4 featured collections on the homepage.  maybe we can condense the surface area of the page and link to the /collections page where we expose more details ?

**Comment by alohr51** [comment]
Okay that sounds good to me, I'll restructure this issue then to create a collections page when we have more details / collections to show

**Comment by mdenby** [comment]
@nateziemann I'm good with leaving the four featured collections on the homepage, but we'll need a collections page too. I'm not sure if you want to go ahead and add that now or wait until we have collection number five.

I imagine the collection should still open to reveal the tools. Later we can add 'view details' button. We can go over this on Friday though when we review 'Getting Started'.
